content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+
+Craft your message so that it is specific, friendly and most importantly accommodating. Make meeting you sound like a very light lift. Here is a message I sent many years ago:
+
+>Hi XXX, I am building out an Analytics team and would love to chat with you to learn about how you've gone about building your team at YYY company. I can imagine you have similar challenges (e.g., interfacing with stakeholders and deciding on the generalist vs specialist model). Happy to work around your schedule and come to you. Let me know when works best.
+
+
+### Step 4: Ask good questions and make a genuine connection
+
+People can quickly tell if you are being disingenuous and only want a job or an introduction. I advise going into the conversation with the goal of learning and potentially forming a professional or personal relationship - a job or an introduction is icing on the cake. I met many of my closest friends in the NYC scene through conversations just like this.
+
+During the conversation, make sure you have a list of topics you want to cover, but allow the conversation to flow naturally. You don't want your coffee chat to feel like an interrogation. Also, if you force the conversation down a specific path you may miss a relevant topic that your coffee mate is especially knowledgeable or passionate about.
+
+
+### Step 5: Ask for recommended contacts (and repeat)
+
+While your first wave of conversations may not be perfectly relevant to the topics you area looking to learn about, your coffee mates may know other, more relevant people to chat with. After a genuine, friendly conversation it is perfectly acceptable to ask your coffee mate if she has anyone you would recommend. You may even find that during a great conversation your coffee mate might even start mentioning people to talk to without you even having to ask.
+
+You will not hit it off with everyone you meet and that is fine - just make sure you are polite and respectful. However, if you continue to work at it you will continue to develop your own analytics expert network and hopefully meet a few friends.
+
+
+## So when do you stop? Never.
+
+I'm in the field of Analytics, because I love it and like any topic I love, I am happy to talk (and write) about it for hours. Whether it is a conversation over lunch or a panel at a conference, I continue to learn from the diverse opinions of my friends and colleagues. While we may not always agree (you should hear Michael and me discuss [reporting]({{< relref "datareporting-is-a-gateway-drug.md" >}})), I always learn.
+
+Your analytics brain trust is especially helpful to stay on top of the latest developments in an evolving field like Analytics. It is impossible to stay on top of every new technology and trend so it have a diverse group of friends with a wide range of interests.
```

Comment:
`so it (pays to?) have a diverse group of friends with a wide range of interests.`